src/main/java/com/reckue/post/services/realizations/PollNodeServiceRealization.java

```diff
@@ -0,0 +1,190 @@
+        } else {
+            throw new ModelAlreadyExistsException("PollNode already exists");
+        }
+    }
+
+    /**
+     * This method is used to update data in an object of class PollNode.
+     * Throws {@link ModelNotFoundException} in case
+     * if such object isn't contained in database.
+     * Throws {@link IllegalArgumentException} in case
+     * if such parameter is null.
+     *
+     * @param node object of class PollNode
+     * @return node object of class PollNode
+     */
+    @Override
+    public PollNode update(PollNode node) {
+        if (node.getId() == null) {
+            throw new IllegalArgumentException("The parameter is null");
+        }
+        if (!pollNodeRepository.existsById(node.getId())) {
+            throw new ModelNotFoundException("PollNode by id " + node.getId() + " is not found");
+        }
+        PollNode savedPollNode = PollNode.builder()
+                .id(node.getId())
+                .title(node.getTitle())
+                .build();
+        return pollNodeRepository.save(savedPollNode);
+    }
+
+    /**
+     * This method is used to get all objects of class PollNode.
+     *
+     * @return list of objects of class PollNode
+     */
+    @Override
+    public List<PollNode> findAll() {
+        return pollNodeRepository.findAll();
+    }
+
+    /**
+     * This method is used to get all objects of class PollNode by parameters.
+     *
+     * @param limit  quantity of objects
+     * @param offset quantity to skip
+     * @param sort   parameter for sorting
+     * @param desc   sorting descending
+     * @return list of objects of class PollNode
+     */
+    @Override
+    public List<PollNode> findAll(Integer limit, Integer offset, String sort, Boolean desc) {
+        if (limit == null) limit = 10;
+        if (offset == null) offset = 0;
+        if (StringUtils.isEmpty(sort)) sort = "id";
+        if (desc == null) desc = false;
+
+        if (limit < 0 || offset < 0) {
+            throw new IllegalArgumentException("Limit or offset is incorrect");
+        }
+        return findAllByTypeAndDesc(sort, desc).stream()
+                .limit(limit)
+                .skip(offset)
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * This method is used to sort objects in descending order by type.
+     *
+     * @param sort parameter for sorting
+     * @param desc sorting descending
+     * @return list of objects of class PollNode sorted by the selected parameter for sorting
+     * in descending order
+     */
+    public List<PollNode> findAllByTypeAndDesc(String sort, boolean desc) {
+        if (desc) {
+            List<PollNode> node = findAllBySortType(sort);
+            Collections.reverse(node);
+            return node;
+        }
+        return findAllBySortType(sort);
+    }
+
+    /**
+     * This method is used to sort objects by type.
+     *
+     * @param sort type of sorting: name, default - id
```

Comment:
You need to correct sorting type - tittle and add another types which you will add to PollNode.